### ISSUE TITLE
Consistently cannot open certain projects.

### ISSUE THREAD

**Issue by denitto** [avocode/avocode#966]
Not sure why, but I cannot open this project in the app:

https://manager.avocode.com/view/3ab4cb4d10754c2b98d6471dade48c7b/

Leads to "An unexpected error has occurred" every time.  

-----
App v.2.8.4
Source UID: 33647
```
TypeError: Cannot read property 'type' of undefined
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:20450
    at arrayMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/lodash/index.js:1406:25)
    at Function.map (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/lodash/index.js:6710:14)
    at e._readLayerFonts (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:20394)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:19674
    at Array.map (native)
    at e._processLayers (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:19433)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:19579
    at Array.map (native)
    at e._processLayers (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:19433)
    at new e (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:17568)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:200
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:10:1532
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:439:3)
```

**Comment by vojtatranta** [comment]
Fixed by GL:!624

**Comment by petrbrzek** [comment]
@denitto Thanks for the report. It has been fixed in our internal release.

**Comment by denitto** [comment]
Woooow that was quick.  Thanks guys.  When is that release(ish)?

**Comment by connor-bean** [comment]
We just released 2.11 so I'm closing this out. Please let us know if this problem persists.

You can download the update at https://avocode.com/news.html

Thanks for your patience on this.

Connor, Avocode Support
